finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+|offset |integer |query |optional |How many transactions to return per page in a response. You can specify `offset` with any integer greater than 0. <br/>If you want to use `limit` and `offset`, don't use `includeTransactionType`. Because `limit` and `offset` will be ignored when `includeTransactionType` is used. 
+</details>
+
+#### Supported transaction types
+> **Note:** Deprecated transaction types are no longer in use, but may appear in historic data.
+<details open="true">
+<summary>Supported transaction types</summary>
+
+|Type |Description
+|:---- |:----
+|ADJUSTMENT |A bookkeeping adjustment.
+|ADVANCE |The cash advance given by Zettle to a merchant. A cash advance is a type of financing that is offered to merchants based on their sales history. The advance is paid back with monthly down payments.
+|ADVANCE_DOWNPAYMENT |A down payment on a previously paid out cash advance.
+|ADVANCE_FEE_DOWNPAYMENT |The netting of a cash advance fee.
+|BANK_ACCOUNT_VERIFICATION (Deprecated) |N/A
+|CARD_PAYMENT |A card payment. Contains a reference to the card payment in the Purchase API.
+|CARD_PAYMENT_FEE |The commission part of a card payment.
+|CARD_PAYMENT_FEE_REFUND |The commission part of a refund.
+|CARD_REFUND |A card refund. <!-- Is it not obvious? -- Accompanied by CARD_PAYMENT_FEE_REFUND. --> Contains a reference to the card payment refund in the Purchase API.
+|CASHBACK |Money given to a merchant to retroactively adjust the card payment fee rate.
+|CASHBACK_PAYOUT (Deprecated) |N/A
+|EMONEY_TRANSFER (Deprecated) |N/A
+|FAILED_PAYOUT |A previous payout transaction has failed and been made void. The payout money is returned to the merchant's Zettle liquid account.
+|FEE_DISCOUNT_REVOCATION |An internal reclaim of an outstanding fee discount that is not consumed within a certain time frame. As these funds are reclaimed from a special fee discount account, the transaction will not be visible in the Zettle liquid account of the merchant.
+|FROZEN_FUNDS |The money that is frozen to cover a chargeback. When the issuing back initiates a chargeback, the money will be removed from the merchant's liquid account and marked as frozen to cover the chargeback. If the chargeback is later revoked, the money will be returned to the merchants liquid account with a new and positive transaction of the same type. It effectively makes the initial FROZEN_FUNDS transaction void.
+|INVOICE_PAYMENT |An invoice payment.
+|INVOICE_PAYMENT_FEE |An invoice payment fee.
+|PAYMENT |An alternative third-party payment method where Zettle handles the funds. For example, PayPal QR code and Klarna QR code.
+|PAYMENT_FEE |The fee for a third-party payment method. For example,PayPal QR code and Klarna QR code.
+|PAYOUT |A payout from the merchant's Zettle liquid account to the merchant’s bank account .  
```

Comment:
Maybe it's no big deal but more and more we will have PayPal users migrating to Zettle, and these won't get the money transferred from liquid account to a bank account, the money will actually go to their PayPal Wallet.